### ISSUE TITLE
Maximum call stack size exceeded error when generating collection from WSDL

### ISSUE THREAD

**Issue by pozil** [a85/POSTMan-Chrome-Extension#10160]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Describe the Issue

Salesforce exposes some WSDL files for generating integrations. I tried to import such a file in Postman to try out the WSDL support. I was able to import the API but I got a "Maximum call stack size exceeded" error when trying to generate a test collection from the API.

### Steps To Reproduce

1. download the attached [wsdl.txt](https://github.com/postmanlabs/postman-app-support/files/6884911/wsdl.txt) file and rename it to `wsdl.xml` (GitHub won't let me upload an XML file)
1. import the WSDL in Postman
1. try to generate a test collection from the imported API
1. notice the "Maximum call stack size exceeded" error

### Screenshots or Videos

![Screen Shot 2021-07-27 at 11 32 53](https://user-images.githubusercontent.com/5071767/127144620-cc3560ef-3933-41d4-96f6-97a9ae4e4809.png)

### Environment Information

```markdown
- Operating System: macOS BigSur (11.4)
- Platform Type: Native App
- Postman Version: 8.9.1 (latest at time of writing)
```

**Comment by VShingala** [comment]
@pozil We have identified the issue and are working on a fix.

**Comment by pozil** [comment]
Awesome, thanks for the update!

**Comment by DawMatt** [comment]
Great to hear! I encountered the same issue today on Windows when importing an ISA-95 OperationsRequest WSDL ( OperationsRequestService_v1_0.wsdl ). Unfortunately I can't attach this file due to the licensing but glad to see a more easily accessible standard can be used to reproduce the issue.

Please note this issue showed up when generating a documentation collection as well, not just when generating test collections.

**Comment by VShingala** [comment]
@pozil @DawMatt This issue is now fixed with the latest Postman App (8.11.0) and web versions. Feel free to reopen this if there are any further issues you are facing.

**Comment by pozil** [comment]
Awesome, thanks for the update.
